lib/veri_report.ml

```diff
@@ -0,0 +1,56 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std 
+
+module Map = String.Map
+
+type hist = (string * int) list [@@deriving bin_io, sexp]
+
+type succ = [ 
```

Comment:
what this `frame -> bool` function will say? Failure or not? We are not interested in solving a decision problem: is BIL correct. We already know the answer. We're solving a search problem - we need to find what is incorrect, and why, and how often. So the `veri` should only record mismatching event. And later, different reporters will generate human readable information from this mismatching sets.